pom.xml

```diff
@@ -292,5 +292,9 @@
 			<id>oss-snapshot</id>
 			<url>https://oss.sonatype.org/content/repositories/snapshots</url>
 		</repository>
+		<repository>
```

Comment:
this is not required as the artifacts are promoted to maven central.

Reply:
http://search.maven.org/#artifactdetails|com.github.camel-tooling|camel-tooling-common|0.0.2|jar